components/VoteTicker/VoteTicker.tsx

```diff
@@ -0,0 +1,130 @@
+import { useState, useEffect } from "react";
+import styled from "styled-components";
+import Clock from "public/assets/clock.svg";
+import UpRightArrow from "public/assets/up-right-arrow.svg";
+import calculateTimeRemaining from "./calculateTimeRemaining";
+
+type theme = "light" | "dark";
+
+interface Props {
+  theme: theme;
+}
+
+const VoteTicker: React.FC<Props> = ({ theme }) => {
```

Comment:
Using [styled components native theming](https://styled-components.com/docs/advanced) will make your life much easier with this

Reply:
I'll dig into this for a potential refactor :)